src/phoenix/server/main.py

```diff
@@ -108,6 +113,28 @@ def _load_items(
         queue.put(item)
 
 
+def _send_spans(spans: Iterable[Span], url: str) -> None:
+    # TODO: Ingest fixtures without networking for read-only deployments
```

Comment:
might want tickets, or at least name the TODO(persistence) so we can do a sweep later

Reply:
ok